lib/orator/index.js

```diff
@@ -327,18 +327,29 @@ class Orator {
           avatarURL
         } = response
 
-        const hook = await findOrCreateManagedWebhook(
-          msg.channel,
-          bot.user.id,
-          this.webhookName
-        )
+        try {
+          const hook = await findOrCreateManagedWebhook(
+            msg.channel,
+            bot.user.id,
+            this.webhookName
+          )
 
-        newMessage = await bot.executeWebhook(hook.id, hook.token, {
-          ...content,
-          username,
-          avatarURL,
-          wait: true
-        })
+          newMessage = await bot.executeWebhook(hook.id, hook.token, {
+            ...content,
+            username,
+            avatarURL,
+            wait: true
+          })
+        } catch (e) {
+          switch (e.message) {
+            case 'Missing Permissions':
+              this.createMessage(bot.user, msg.channel,
+                'Missing Webhook Permissions')
+              break
+          }
+
+          throw e
```

Comment:
since we want to actually continue this flow, let's only throw on the `default` switch case. And lets log a `warn` saying "Missing Webhook Permissions" in the case of Missing Permissions above